### DISCUSSION TITLE
Query Input Displayed in Console

### DISCUSSION THREAD

**Discussion by cbath** [jshinonome/vscode-q#35] [Ideas]
Hey, really love (rely) on this great plug-in!

Something that helps develpment is being able to copy input/ouput from the console to Slack. An option to display the code ran in the console before the output would be amazing.  E.g. 

```q
>>> q-read-q-read-company @ 12:25:10 ---- 3(ms) elapsed
query: 2 * 6

12
<<<
```

**Comment by jshinonome** [comment]
Thank your for using this plugin :)

I was thinking of adding input. When you sent a huge query, e.g. a 1000 chars' query, how do you want it to display the query? How abou a configuration of width and length for input?

**Comment by cbath** [comment]
That’d be amazing. Normally when we’re devving on linux each developer will have their own favorite combination for “\c”  for setting length + width, that customizing that would be great.

Thanks again Jo! I’ll make sure to point the other fans on my team to your “buy me a coffee” link 😊

**Reply by jshinonome** [reply]
You are welcome, Bath:)

I will try to get this done by this week.

And thank you for recommending this plugin to your team~

**Comment by jshinonome** [comment]
I added two options in 2.2.1

![image](https://user-images.githubusercontent.com/64458591/115959001-eac3ed00-a544-11eb-9924-61951d5e93a3.png)

Let me know if it works.

**Comment by cbath** [comment]
Looks great Jo! This will genuinely save me a few minutes a day!

Thank you so much,

Craig.

**Reply by jshinonome** [reply]
You are welcome :)
